app/lib/plugins/settings/index.js

```diff
@@ -1,5 +1,4 @@
 import get from './get'
 import validate from './validate'
-import add from './add'
 
-export default { get, validate, add }
```

Comment:
@BrainMaestro but what do you mean by add to config? We save changes values to global config [here](https://github.com/KELiON/cerebro/blob/7d8b75702e79668b719afef1aa46f295a6d4c13e/app/main/plugins/core/cerebro/plugins/Preview/Settings.js#L39)